### PR TITLE
fix: update shadow db limitation for cloud db

### PR DESCRIPTION
**Pull request by Jolg42**
Outdated content needed a refresh

Preview at
https://deploy-preview-2366--prisma2-docs.netlify.app/docs/concepts/components/prisma-migrate/shadow-database#shadow-database-user-permissions

Live at
https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#shadow-database-user-permissions

### CODE REVIEW

**Review by tomhoule — APPROVED** [review]
Seems like the right minimal fix, nice :+1:

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 1]
```diff
@@ -124,8 +124,7 @@ Database error: Error querying the database: db error: ERROR: permission denied
 To resolve this error:
 
 - If you are working locally, we recommend that you change the database user's privileges.
-- If you are developing against a cloud-based database (for example, on Heroku), we recommend that you change to a local web environment (such as running your development database on Docker) for the time being and follow the discussions on [GitHub issue #4571](https://github.com/prisma/prisma/issues/4571).
-- If you are developing against a cloud-based database (for example, on Heroku) and **cannot** switch, please remain on version 2.12.0 and follow the discussions on [GitHub issue #4571](https://github.com/prisma/prisma/issues/4571).
+- If you uare developing against a cloud-based database see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually)
```
Comment (tomhoule) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]: typo: `uare`

Comment (Jolg42) [reply content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]:
```suggestion
- If you are developing against a cloud-based database see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually)
```

File: content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx [hunk 2]
```diff
@@ -124,8 +124,7 @@ Database error: Error querying the database: db error: ERROR: permission denied
 To resolve this error:
 
 - If you are working locally, we recommend that you change the database user's privileges.
-- If you are developing against a cloud-based database (for example, on Heroku), we recommend that you change to a local web environment (such as running your development database on Docker) for the time being and follow the discussions on [GitHub issue #4571](https://github.com/prisma/prisma/issues/4571).
-- If you are developing against a cloud-based database (for example, on Heroku) and **cannot** switch, please remain on version 2.12.0 and follow the discussions on [GitHub issue #4571](https://github.com/prisma/prisma/issues/4571).
+- If you are developing against a cloud-based database see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually)
```
Comment (Jolg42) [inline content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx]:
```suggestion
- If you are developing against a cloud-based database (for example, on Heroku) see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually)
```

**Pull request merged**